lib/galaxy_test/api/test_histories.py

```diff
@@ -77,6 +77,15 @@ def test_show_history(self):
         self._assert_has_keys(state_details, *states)
         self._assert_has_keys(state_ids, *states)
 
+    def test_show_history_returns_expected_urls(self):
+        # This test can be dropped when the URL attributes become deprecated
+        history_id = self._create_history("TestHistoryForUrls")["id"]
+        show_response = self._show(history_id)
+        self._assert_has_key(show_response, 'id', 'url', 'contents_url')
+
+        assert history_id in show_response["url"]
+        assert history_id in show_response["contents_url"]
```

Comment:
These won't fail for "weird" URLs since the `history_id` is contained in those as well.

Reply:
Good catch!

Reply:
Good point! It should be fixed now, thank you!